src/main/java/com/jcabi/github/User.java

```diff
@@ -239,6 +268,16 @@ public UserEmails emails() {
         }
 
         @Override
+        public List<Notification> notifications() throws IOException {
+            return this.user.notifications();
+        }
+
+        @Override
+        public void markAsRead(final Date lastread) {
+            // Will be implemented later
```

Comment:
Wouldn't it be enough to just call this.user.markAsRead(lastread) ?
If so, then we could remove the todo here, and add the one in the `User`, mentioned above